logging/tests/unit/handlers/test_app_engine.py

```diff
@@ -24,34 +25,44 @@ def _get_target_class(self):
         return AppEngineHandler
 
     def _make_one(self, *args, **kw):
-        import tempfile
+        return self._get_target_class()(*args, **kw)
 
-        from google.cloud._testing import _Monkey
-        from google.cloud.logging.handlers import app_engine as _MUT
+    def test_constructor(self):
+        import mock
```

Comment:
You can import `mock` at module level if you like (no need for a change though if you don't want to)